schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE ANIMALS (
+    id INT,
```

Comment:
## [OPTIONAL]
- [ ] Kindly I would suggest you autogenerate the id in the animal's table, you can do it buy adding this config to the id column :point_down: 

```sql
id INT GENERATED ALWAYS AS IDENTITY,
```
with this configuration you no longer need to hard code the value of the id when insterting data in the tabe.

Reply:
Thank you, I will surely implement it 💯